src/main/java/org/spongepowered/common/service/pagination/ListPagination.java

```diff
@@ -64,7 +65,9 @@ public ListPagination(MessageReceiver src, PaginationCalculator calc, List<Map.E
 
     @Override
     protected Iterable<Text> getLines(int page) throws CommandException {
-        if (page < 1) {
+        if (this.pages.size() == 0) {
+            return Lists.newArrayList();
```

Comment:
Assuming this is to be used to read-only: `return ImmutableList.of()` - uses a single instance